code/reports/country/country-report.Rmd

```diff
@@ -5,13 +5,13 @@ params:
   location_name: "Overall"
   plot_weeks: 4
   data_weeks: 10
+  config: "https://raw.githubusercontent.com/epiforecasts/covid19-forecast-hub-europe/main/forecasthub.yml"
```

Comment:
Should this be a local path? Don't feel strongly but it seems that if e.g. I wanted to change the config in a branch I'm working on then this should filter through to these.

Reply:
It's just the default value. You can change it for a local path when you are developing. In production, we need a URL because the reports will live in the `-website` repo. A local path cannot work with GitHub actions.

If you wish, I can keep a local path as default and just put the URL in the `compile-...-reports.r` script.

Reply:
No, this makes sense.